stdlib/test/builtin/test_print.mojo

```diff
@@ -72,8 +72,13 @@ fn test_print_sep():
     # CHECK: a/1/2xx
     print("a", 1, 2, sep="/", end="xx\n")
 
+fn test_print_stderr():
+    # TODO: check print to stderr
+    print("stderr", file=2)
```

Comment:
To avoid magic numbers, I wonder if we can add `alias stderr = 2` in `sys`, same for `stdout`.
With this, we can do `print("something", file=sys.stderr)`. Later on, when we have mutable global variables, we can replace those numbers by actual file descriptors.

Reply:
I agree, we should have named constants for these, which would have better readability and consistency with Python.

I suggest we add a new file called `sys/_io.mojo` and add these constants there:

```mojo
alias stdout: Int = 1
alias stderr: Int = 2
```

These can be integer values for now, and we'll plan to adjust them to be "file objects" over time as the IO parts of the standard library shape up.

Reply:
done.